.travis.yml

```diff
@@ -105,11 +105,11 @@ matrix:
         # - export RAY_REDIS_SERVER_VALGRIND=1
 
         # # Python3.5+ only. Otherwise we will get `SyntaxError` regardless of how we set the tester.
-        - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest --durations=5 python/ray/experimental/test/async_test.py
-        - python -m pytest --durations=5 python/ray/tests/test_mini.py
-        - python -m pytest --durations=5 python/ray/tests/test_array.py
-        - python -m pytest --durations=5 python/ray/tests/test_multi_node_2.py
-        - python -m pytest --durations=5 python/ray/tests/test_node_manager.py
+        - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest -v --durations=5 python/ray/experimental/test/async_test.py
+        - python -m pytest -v --durations=5 python/ray/tests/test_mini.py
+        - python -m pytest -v --durations=5 python/ray/tests/test_array.py
+        - python -m pytest -v --durations=5 python/ray/tests/test_multi_node_2.py
+        - python -m pytest -v --durations=5 python/ray/tests/test_node_manager.py
```

Comment:
Can we wrap these in `ci/suppress_output` then?

It will hide the durations though, but will print output if the test hangs.

I think there is also a way to set a global timeout in pytest: https://pypi.org/project/pytest-timeout/1.2.1/

Reply:
`ci/suppress_output` will also hide the progress. Considering `-v` only adds 450-ish lines, I think that is fine?
And the global timeout is a good idea.

Reply:
Ok, let's see how it goes. I think this might be ok (mobile browsers seems to start choking only at a couple thousand lines).